python/shotgun_menus/entity_field_menu.py

```diff
@@ -78,6 +78,10 @@ def __init__(self, sg_entity_type, parent=None, bg_task_manager=None):
 
         :param bg_task_manager: The task manager the menu will use if it needs to run a task
         :type bg_task_manager: :class:`~task_manager.BackgroundTaskManager`
+
+        :param int project_id: The project entity id to use when querying fields lists. If
+                               None, the current context's project will be used if one is
+                               set, otherwise the "site" level fields will be queried.
```

Comment:
i thin this is similar to this comment:

https://github.com/shotgunsoftware/tk-framework-shotgunutils/pull/46#discussion_r71131214

If project id is None, aren't we using the site wide settings?